.azdevops/image-pipeline.yml

```diff
@@ -42,16 +39,105 @@ jobs:
       VERSION: $(Build.BuildId)
 
     # docker push
+  - script: 'make push'
+    workingDirectory: '$(modulePath)'
+    displayName: 'make push (docker push)'
+    env:
+      VERSION: $(Build.BuildId)
+      DOCKER_PASS: $(DOCKER_PASS)
+
+- job: Deploy

```

Comment:
Is this the name of the job?  I think this would make more sense as "e2e-tests"?